### ISSUE TITLE
Crash (1.7.10)

### ISSUE THREAD

**Issue by Fanofkatyperry** [OpenMods/OpenBlocks#570]
The game keeps crashing when trying to load. This is the crash report I get:
```
---- Minecraft Crash Report ----
// I let you down. Sorry :(

Time: 1/1/15 4:41 PM
Description: There was a severe problem during mod loading that has caused the game to fail

cpw.mods.fml.common.LoaderException: java.lang.NoSuchMethodError: net.minecraftforge.client.ClientCommandHandler.registerCommand(Lnet/minecraft/command/ICommand;)Lnet/minecraft/command/ICommand;
	at cpw.mods.fml.common.LoadController.transition(LoadController.java:162)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:515)
	at cpw.mods.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:239)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:480)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:867)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:134)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.NoSuchMethodError: net.minecraftforge.client.ClientCommandHandler.registerCommand(Lnet/minecraft/command/ICommand;)Lnet/minecraft/command/ICommand;
	at openmods.proxy.OpenClientProxy.preInit(OpenClientProxy.java:77)
	at openmods.OpenMods.preInit(OpenMods.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:513)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:208)
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:187)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:118)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:513)
	... 10 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows Vista (x86) version 6.0
	Java Version: 1.7.0_60, Oracle Corporation
	Java VM Version: Java HotSpot(TM) Client VM (mixed mode), Oracle Corporation
	Memory: 105834088 bytes (100 MB) / 269320192 bytes (256 MB) up to 523501568 bytes (499 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx512M -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.25.1190 Minecraft Forge 10.13.0.1190 6 mods loaded, 6 mods active
	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	FML{7.10.25.1190} [Forge Mod Loader] (forge-1.7.10-10.13.0.1190.jar) Unloaded->Constructed->Pre-initialized
	Forge{10.13.0.1190} [Minecraft Forge] (forge-1.7.10-10.13.0.1190.jar) Unloaded->Constructed->Pre-initialized
	OpenModsCore{0.6} [OpenModsCore] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	OpenMods{0.6} [OpenMods] (OpenModsLib-1.7.10-0.6-snapshot-283-deobf.jar) Unloaded->Constructed->Errored
	OpenBlocks{1.3} [OpenBlocks] (OpenBlocks-1.7.10-1.3-snapshot-531.jar) Unloaded->Constructed->Errored
	OpenModsLib crash transformers: [gl_capabilities_hook:FINISHED],[player_render_hook:ENABLED],[map_gen_fix:ENABLED],[movement_callback:FINISHED],[stencil_patches:FINISHED]
```

**Comment by Parker8283** [comment]
You're on a REALLY out of date version of Forge. Update.

**Comment by Fanofkatyperry** [comment]
Oh, ok, I didnt know u had to update forge, thanks!

**Comment by Fanofkatyperry** [comment]
it still doesnt work! ---- Minecraft Crash Report ----
// Don't do that.

Time: 1/1/15 4:53 PM
Description: Initializing game

java.lang.NoSuchFieldError: depthBuffer
	at openmods.stencil.FramebufferHooks.attachRenderbuffer(FramebufferHooks.java:32)
	at net.minecraft.client.shader.Framebuffer.func_147605_b(Framebuffer.java:122)
	at net.minecraft.client.shader.Framebuffer.func_147613_a(Framebuffer.java:56)
	at net.minecraft.client.shader.Framebuffer.<init>(Framebuffer.java:37)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:467)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:867)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at openmods.stencil.FramebufferHooks.attachRenderbuffer(FramebufferHooks.java:32)
	at net.minecraft.client.shader.Framebuffer.func_147605_b(Framebuffer.java:122)
	at net.minecraft.client.shader.Framebuffer.func_147613_a(Framebuffer.java:56)
	at net.minecraft.client.shader.Framebuffer.<init>(Framebuffer.java:37)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:467)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:867)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows Vista (x86) version 6.0
	Java Version: 1.7.0_60, Oracle Corporation
	Java VM Version: Java HotSpot(TM) Client VM (mixed mode), Oracle Corporation
	Memory: 89209224 bytes (85 MB) / 182132736 bytes (173 MB) up to 523501568 bytes (499 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx512M -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: 
	Launched Version: 1.7.10-Forge10.13.2.1277
	LWJGL: 2.9.1
	OpenGL: GeForce 6150SE nForce 430/PCI/SSE2/3DNOW! GL version 2.1.2, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because EXT_framebuffer_object is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: [Sphax PureBDcraft  32x MC18.zip]
	Current Language: ~~ERROR~~ NullPointerException: null
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: Off (1

**Comment by boq** [comment]
Somehow you have downloaded deobfuscated jar. Just download files from openmods.info
